### PR TITLE
Fix warnings on Visual Studio

### PR DESCRIPTION
**Pull request by chosungmann**

### CODE REVIEW

File: src/layer/yolov3detectionoutput.cpp [hunk 1]
```diff
@@ -168,7 +168,7 @@ int Yolov3DetectionOutput::forward(const std::vector<Mat>& bottom_blobs, std::ve
     std::vector<BBoxRect> all_bbox_rects;
     std::vector<float> all_bbox_scores;
 
-    for (size_t b = 0; b < bottom_blobs.size(); b++)
+    for (int b = 0; b < bottom_blobs.size(); b++)
```
Comment (nihui) [inline src/layer/yolov3detectionoutput.cpp]: ooop ?

File: src/layer/detectionoutput.cpp [hunk 2]
```diff
@@ -103,14 +103,14 @@ static void qsort_descent_inplace(std::vector<T>& datas, std::vector<float>& sco
...
 
 static void nms_sorted_bboxes(const std::vector<BBoxRect>& bboxes, std::vector<int>& picked, float nms_threshold)
 {
     picked.clear();
 
-    const int n = bboxes.size();
+    const size_t n = bboxes.size();
 
     std::vector<float> areas(n);
     for (int i = 0; i < n; i++)
```
Comment (nihui) [inline src/layer/detectionoutput.cpp]: signed and unsigned compare here and the following loops ?

File: src/layer/yolodetectionoutput.cpp [hunk 3]
```diff
@@ -128,14 +128,14 @@ static void qsort_descent_inplace(std::vector<T>& datas, std::vector<float>& sco
...
 
 static void nms_sorted_bboxes(const std::vector<BBoxRect>& bboxes, std::vector<int>& picked, float nms_threshold)
 {
     picked.clear();
 
-    const int n = bboxes.size();
+    const size_t n = bboxes.size();
 
     std::vector<float> areas(n);
     for (int i = 0; i < n; i++)
```
Comment (nihui) [inline src/layer/yolodetectionoutput.cpp]: signed and unsigned compare here and the following loops ?

File: src/layer/yolov3detectionoutput.cpp [hunk 4]
```diff
@@ -115,14 +115,14 @@ static void qsort_descent_inplace(std::vector<T>& datas, std::vector<float>& sco
...
 
 static void nms_sorted_bboxes(const std::vector<BBoxRect>& bboxes, std::vector<int>& picked, float nms_threshold)
 {
     picked.clear();
 
-    const int n = bboxes.size();
+    const size_t n = bboxes.size();
 
     std::vector<float> areas(n);
     for (int i = 0; i < n; i++)
```
Comment (nihui) [inline src/layer/yolov3detectionoutput.cpp]: signed and unsigned compare here and the following loops ?

File: src/net.cpp [hunk 5]
```diff
@@ -972,7 +972,7 @@ int Net::find_layer_index_by_name(const char* name) const
 
 int Net::custom_layer_to_index(const char* type)
 {
-    const int custom_layer_registry_entry_count = custom_layer_registry.size();
+    const size_t custom_layer_registry_entry_count = custom_layer_registry.size();
     for (int i=0; i<custom_layer_registry_entry_count; i++)
```
Comment (nihui) [inline src/net.cpp]: signed and unsigned compare here ?

File: tools/mxnet/mxnet2ncnn.cpp [hunk 6]
```diff
@@ -762,7 +762,7 @@ static bool read_mxnet_param(const char* parampath, std::vector<MXNetParam>& par
 
 static void fuse_shufflechannel(std::vector<MXNetNode>& nodes, std::vector<MXNetParam>& params, std::map<int, int>& node_reference, std::set<std::string>& blob_names, int& reduced_node_count)
 {
-    int node_count = nodes.size();
+    size_t node_count = nodes.size();
     for (int i=0; i<node_count; i++)
```
Comment (nihui) [inline tools/mxnet/mxnet2ncnn.cpp]: signed and unsigned compare here ?

File: src/layer/yolov3detectionoutput.cpp (hunk 1 above)
Comment (chosungmann) [reply src/layer/yolov3detectionoutput.cpp]: done

File: src/layer/yolov3detectionoutput.cpp (hunk 4 above)
Comment (chosungmann) [reply src/layer/yolov3detectionoutput.cpp]: done

File: src/layer/yolodetectionoutput.cpp (hunk 3 above)
Comment (chosungmann) [reply src/layer/yolodetectionoutput.cpp]: done

File: src/layer/detectionoutput.cpp (hunk 2 above)
Comment (chosungmann) [reply src/layer/detectionoutput.cpp]: done

File: src/net.cpp (hunk 5 above)
Comment (chosungmann) [reply src/net.cpp]: done

File: tools/mxnet/mxnet2ncnn.cpp (hunk 6 above)
Comment (chosungmann) [reply tools/mxnet/mxnet2ncnn.cpp]: done

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1431) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1431) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=1431) it.</sub>

File: src/layer/yolov3detectionoutput.cpp (hunk 1 above)
Comment (nihui) [reply src/layer/yolov3detectionoutput.cpp]: It seems to be unchanged and not resolved

Comment (chosungmann) [reply src/layer/yolov3detectionoutput.cpp]: Oops, sorry about that. I've missed some commits during interative rebase. Done.

**Pull request merged**